### PR TITLE
Link to CLI profile generator added

### LINKED ISSUE #2206: [DOCS] Add Linkfree-CLI project to ReadMe file
### Description

Adding @Pradumnasaraf 's Linkfree profile generator in CLI will be great. 
The link to repo can be found here: https://github.com/Pradumnasaraf/LinkFree-CLI

Suggestions discussed on live stream:

- Add a *note* above the existing note with the following content: (you can use your content)
"You can generate the code for your profile using Pradumna's Linkfree-CLI profile generator." 

You can also look at a parallel issue raised here to add Tom's repo: #2207 
(Don't forget to add the link to the repo)

image

### PR DESCRIPTION
**Pull request by krupalitrivedi**
## Fixes Issue

closes #2206 

## Changes proposed

The link to Pradumna's CLI profile generator for Linkfree has been added.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

image

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2572"></a>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +72,8 @@ Looking for inspiration? You can view the following profiles for an example:
 
 Your brand new file should look something similar to this one:
 
+> **Note** **CLI Profile Generator**: To generate the code for your profile using CLI, visit [Pradumna's Linkfree-CLI profile generator](https://github.com/Pradumnasaraf/LinkFree-CLI)
```
Comment (eddiejaoude) [inline README.md:75]: I think @Pradumnasaraf's project is included in LinkFree, maybe we could also have steps to how to use it and link for further information.

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
The Changelog and package files I don't think are needed in this PR (check your branch is up to date)

**Comment by krupalitrivedi** [comment]
Sure, Eddie, let me work on the further changes and make a pull request, feel free to close this one. (I did update my branch before making changes but there seems to be some problem)

**Comment by eddiejaoude** [comment]
It is up to you if you want to close it and open a new one - you should be able to close it if you wish

**Comment by krupalitrivedi** [comment]
Closing this PR - will open another one keeping the discussion here in mind.

**Pull request closed without merging**